utilities/console.py

```diff
@@ -26,14 +31,16 @@ def __init__(self, vm, username=None, password=None, timeout=30, prompt=None):
         Examples:
             from utilities import console
             with console.Console(vm=vm) as vmc:
-                vmc.sendline('some command)
-                vmc.expect('some output')
+                vmc.child.sendline('some command)
+                vmc.child.expect('some output')
```

Comment:
Do you mean this?
```
vmc.expecter.sendline('some command')
vmc.expecter.expect('some output')
```

Reply:
Ahh yes, the rebase failed me. Thanks!
Done